app/controllers/issues_controller.rb

```diff
@@ -137,7 +137,7 @@ def set_columns
     rtp_default_fields = rtp ? rtp.issue_fields.default.field_names : []
 
     @default_columns = rtp_default_fields.presence || default_field_names
-    @all_columns = ['Title'] | rtp_default_fields | dynamic_fields | extra_field_names
+    @all_columns = ['Title'] | rtp_default_fields | default_field_names | dynamic_fields | extra_field_names
```

Comment:
Previously we removed this under the assumption "everything would get brought in via the dynamic fields" but it turns out we were wrong. Tags and Affected were considered default fields, but not included as extra fields.